### PR TITLE
Update serverless-ai-tutorial.md

### PR DESCRIPTION
**Pull request by sohanmaheshwar**
Made some changes to the post including couple of errors in the code that caused it to break:

1. `max_tokens` should be `maxTokens`
2. the `/static` static path was causing the file `dynamic.js` to not be found
3. Added the code for `index.html` and `dynamic.js` in the tutorial 
3. the TS example said `http-js` so made the change the `http-ts`
4. `ai_models = ["llama2-chat"]` was missing in the `spin.toml` file
5. Changed the title to reflect the use case discussed in the post - similar to our other tutorials
6. @rajatjindal suggested adding a link to the repo for people who want to `git clone` the example directly. Added that too 

Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [ ] The `title, `template`, and `date` are all set
- [ ] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [ ] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [ ] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)
- [ ] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [ ] Headings are using Title Case
- [ ] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [ ] Have tested with `npm run test` and resolved all errors
- [ ] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

File: content/spin/serverless-ai-tutorial.md [hunk 1]
```diff
@@ -1,4 +1,884 @@
-title = "Serverless AI Tutorial"
+
```
Comment (karthik2804) [inline content/spin/serverless-ai-tutorial.md:1]: Uneccessary new line

File: content/spin/serverless-ai-tutorial.md [hunk 2]
```diff
@@ -1,4 +1,884 @@
...
+## Conclusion
+
+We want to get feedback on the Serverless AI API. We are curious about what models you would like to use and what applications you are building using Serverless AI. Let us know what you need, and how Fermyon's Serverless AI could potentially help solve a problem for you. We would love to help you write your new Serverless AI application.
+
+## Next Steps
+
+- Try the numerous Serverless AI examples in our GitHub repository called [ai-examples](https://github.com/fermyon/ai-examples).
+- [Contribute](https://developer.fermyon.com/hub/contributing) your Serverless AI app to our [Spin Up Hub](https://developer.fermyon.com/hub).
+- Ask questions and share your thoughts in [our Discord community](https://discord.gg/AAFNfS7NGf).title = "Serverless AI Tutorial"
 template = "spin_main"
```
Comment (karthik2804) [inline content/spin/serverless-ai-tutorial.md:882]: Seems to be a stray copy of the front matter.

**Review by karthik2804 — CHANGES_REQUESTED** [review]

File: content/spin/serverless-ai-tutorial.md
Comment (tpmccallum) [inline content/spin/serverless-ai-tutorial.md:1]: Hi @sohanmaheshwar 
It looks like the whole of the document’s contents has been duplicated twice in one page.
For example, the preview shows the page twice (including the embedded video etc) if you just keep scrolling you will see.

**Review by tpmccallum — CHANGES_REQUESTED** [review]

File: content/spin/serverless-ai-tutorial.md
Comment (sohanmaheshwar) [reply content/spin/serverless-ai-tutorial.md:1]: Hm not sure what happened there. Made the edits, and also removed the Private beta link as we're GA now. Added a line about Spin v1.5 and above.

**Comment by sohanmaheshwar** [comment]
Folks, closing this PR. When other edits were made to this tutorial the file name was changed and that has caused some merge issues. i've consolidated the changes mentioned in this PR to a new PR here #916

**Pull request closed without merging**